app/routes/howcanihelppage.$.tsx

```diff
@@ -0,0 +1,26 @@
+import {useParams} from '@remix-run/react'
```

Comment:
check [this](https://remix.run/docs/en/main/discussion/routes). Instead of having per subpage components, you create per subroute route files

Reply:
I think I understood you. I updated it to use the _index and subroutes, but feel free to change it if this isn't what you mean.